### PR TITLE
remixd modal content updated

### LINKED ISSUE #1241: `connect to localhost` modal should also inform about hardhat ws plugin connection

### PR DESCRIPTION
**Pull request by Aniket-Engg**
fixes #1241

### CODE REVIEW

**Review by ryestew — COMMENTED** [review]
In terms of what is important on this modal:
1. The top needs to have a link to the docs. (done)
2. The bottom needs to have the warning about updating to the latest - especially because it is bridging the browser - filesystem barrier. (could be more emphatic - but done!)
3. The shortcut to the remixd command (done)

There is this section: On successful connection, a session will be started between http://localhost:8080 and your local file system at ws://127.0.0.1:65520 and files will be loaded under localhost workspace in the File Explorers.
Please note, if shared folder is a hardhat project, an additional hardhat websocket plugin will be listening at ws://127.0.0.1:65522 

And I'm not sure any of it is that important - so should it be taken out and just put int the documentation?  (I may be wrong about this)

And there is this bland warning about backing up the folder.  Is anyone really going to do that because of this warning?? And  I'm not sure this is even possible.  I mean how are you going to keep an up to date backup of a folder while you are working on it....OK I mean we could say make sure you have this backed up in git.  Is that better?  Or we can just leave it and told them we told them we had warned them....

In the rest of my review - I'll just suggest text changes.  But we could think about taking out some of the content as noted above.

File: apps/remix-ide/src/app/files/remixd-handle.js [hunk 1]
```diff
@@ -130,25 +131,28 @@ export class RemixdHandle extends WebsocketPlugin {
...
       </div>
-      <div class=${css.dialogParagraph}>If you are just looking for the remixd command here it is:
-        <br><br><b>remixd -s absolute-path-to-the-shared-folder --remix-ide your-remix-ide-URL-instance</b>
+      <div class=${css.dialogParagraph}>If you are just looking for the remixd command, here it is:
+        <br><br><b>${commandText}</b>
         <span class="">${copyToClipboard(() => commandText)}</span>
       </div>
-      <div class=${css.dialogParagraph}>A connection will start a session between <em>${window.location.origin}</em> and your local file system <i>ws://127.0.0.1:65520</i>
-        <br>To see that a connection has been made, check that there is a localhost section in the Files Explorer
+      <div class=${css.dialogParagraph}>On successful connection, a session will be started between <em>${window.location.origin}</em> and your local file system at <i>ws://127.0.0.1:65520</i>
```
Comment (ryestew) [inline apps/remix-ide/src/app/files/remixd-handle.js:145]: When connected, a session will start between

File: apps/remix-ide/src/app/files/remixd-handle.js [hunk 2]
```diff
@@ -130,25 +131,28 @@ export class RemixdHandle extends WebsocketPlugin {
...
+         and files will be loaded under localhost workspace in the File Explorers.
```
Comment (ryestew) [inline apps/remix-ide/src/app/files/remixd-handle.js:146]: The shared folder will be in the File Explorers workspace named "localhost".

File: apps/remix-ide/src/app/files/remixd-handle.js [hunk 3]
```diff
@@ -130,25 +131,28 @@ export class RemixdHandle extends WebsocketPlugin {
...
-        <br><br><b>remixd -s absolute-path-to-the-shared-folder --remix-ide your-remix-ide-URL-instance</b>
+      <div class=${css.dialogParagraph}>If you are just looking for the remixd command, here it is:
+        <br><br><b>${commandText}</b>
         <span class="">${copyToClipboard(() => commandText)}</span>
       </div>
-      <div class=${css.dialogParagraph}>A connection will start a session between <em>${window.location.origin}</em> and your local file system <i>ws://127.0.0.1:65520</i>
-        <br>To see that a connection has been made, check that there is a localhost section in the Files Explorer
+      <div class=${css.dialogParagraph}>On successful connection, a session will be started between <em>${window.location.origin}</em> and your local file system at <i>ws://127.0.0.1:65520</i>
+         and files will be loaded under localhost workspace in the File Explorers. 
+        <br/>Please note, if shared folder is a hardhat project, an additional hardhat websocket plugin will be listening at <i>ws://127.0.0.1:65522</i>
```
Comment (ryestew) [inline apps/remix-ide/src/app/files/remixd-handle.js:147]: Note, if the shared folder is a Hardhat project, an additional Hardhat websocket plugin will be listening at

File: apps/remix-ide/src/app/files/remixd-handle.js (hunk 1 above)
Comment (Aniket-Engg) [reply apps/remix-ide/src/app/files/remixd-handle.js:145]: done

File: apps/remix-ide/src/app/files/remixd-handle.js (hunk 2 above)
Comment (Aniket-Engg) [reply apps/remix-ide/src/app/files/remixd-handle.js:146]: done

File: apps/remix-ide/src/app/files/remixd-handle.js (hunk 3 above)
Comment (Aniket-Engg) [reply apps/remix-ide/src/app/files/remixd-handle.js:147]: updated

**Comment by LianaHus** [comment]
Screenshot from 2021-06-07 08-55-06

**Review by LianaHus — APPROVED** [review]

**Pull request merged**